.github/workflows/modern-ci.yml

```
@@ -0,0 +1,205 @@
+name: Modern CI (Simplified)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)